src/init/starship.ps1

```diff
@@ -123,6 +123,10 @@ $null = New-Module starship {
             "--jobs=$($jobs)"
         )
 
+        if ($Host.UI.RawUI.CursorPosition.Y -eq 0 -or $script:firstRender) {
```

Comment:
Hi @davidkna, I'm not sure if "reviewing clearing" (https://github.com/starship/starship/pull/5072#discussion_r1274604551) also refers to this line. Hopefully this is not seen as ignoring your comment.

Please also consider that starship currently uses [CursorPosition.Y in the transient prompt code](https://github.com/starship/starship/blob/c88aeeaf15acd95df649cb07c167cdd0056c2d1c/src/init/starship.ps1#L89C124-L89C155). So there is some precedent for querying the cursor position 👍